adapters/repos/db/inverted/searcher_ref_filter.go

```diff
@@ -245,3 +241,13 @@ func (r *refFilterExtractor) validate() error {
 
 	return nil
 }
+
+func determineRefLimit() int {
```

Comment:
so basically by default limit is now set to `config.DefaultQueryReferenceLimit = 100k`, "unlimited" when env var explicitly set to negative value, or env var value when it is >= 0?

Reply:
Currently, if env var is set to 0, no refs will be returned